spec/integration/v2/tailorings_spec.rb

```diff
@@ -91,6 +91,35 @@
         run_test!
       end
     end
+
+    post 'Create a Tailoring' do
+      let(:policy_id) do
+        FactoryBot.create(
+          :v2_policy,
+          account: user.account,
+          profile: FactoryBot.create(:v2_profile, ref_id_suffix: 'foo', supports_minors: [1])
+        ).id
+      end
+
+      v2_auth_header
+      tags 'Policies'
+      description 'Create a Tailoring with the provided attributes (for ImageBuilder only)'
```

Comment:
Minor: considering a reference to ImageBuilder 

```suggestion
      description 'Create a Tailoring with the provided attributes (for <a href="https://console.redhat.com/insights/image-builder">ImageBuilder</a> only)'
```

Reply:
Not sure if that's a good idea...